contents/blog/how-to-start-a-growth-team.md

```diff
@@ -0,0 +1,110 @@
+
+## Background
+
+Let's Do This is a marketplace for discovering and signing up for endurance events such as marathons or races. In 2020, a new VP of product joined the team. One of their new initiatives was to create a growth team focusing on user acquisition and retention. Thomas, who was working as a software engineer on another team, was intrigued by this. He previously had some experience working in SEO and marketing but not in *growth* per se. Nonetheless, he jumped at the opportunity, and after a chat with the VP of product, Thomas joined the newly formed growth team.
+
+The team consisted of a few engineers, a product designer, and a data scientist, but no one really had any growth experience either. Their first job was to hire a growth PM to lead the team, which they soon did. However, two months after hiring this person, it was clear they weren't a good fit and were let go. Noticing a gap in the team, Thomas stepped up to become the de facto "half PM - half tech lead" of the team.
+
+Since Thomas had no prior growth experience, his first step was figuring out what to do. To do this, he tapped into his network and had calls with growth experts at other startups.
+
+The advice from them was consistent:
+
+## 1. The answer is in your data
+
+"It's all about finding the right data," Thomas was told. Taking this to heart, they built dashboards to monitor essential metrics. These dashboards tracked channels, funnels, and user flows on the website. They used these dashboards to answer questions such as:
+
+- Where are bookings coming from? 
+- What are the most common user journeys? 
+- Where do users drop off in the journey? 
+- What features are the most engaging for our users? 
+
+Once they had this data, their next focus was on finding the significant drop-offs in their user journeys and focusing on improving them. This led Thomas to the next piece of advice:
+
+## 2. Talk to your customers and build what they want
+
+Once they knew where the drop-offs were, the next step was figuring out why they were happening and what they could do about it. 
+
+In the case of Let's Do This, their target metric was *number of bookings*. They noticed a large drop-off in the signup page during the booking flow and decided to investigate. They sent emails to the customers who dropped off and offered them Amazon gift cards in exchange for a 30-minute call.
+
+They learned from these calls that booking events like marathons is a **big deal** for their customers (they need to train for months for an event!), and they needed to be confident that the event would be right for them. Thus people would often hesitate when booking events. 
+
+Thomas and his team then brainstormed solutions around this. They came up with features to improve trust signals on the event page, as well as adding an "event save" flow that let users save an event and receive reminders about it (without having to register).
+
+## 3. Experiment everything
```

Comment:
```suggestion
## 3. Test everything
```

Reply:
Alt: 
- Experiment on everything
- Experiment with everything

Reply:
Thanks. I prefer "experiment" to "test", since Test could mean checking for bugs, which is not what they were doing.

"Experiment on everything" and "Experiment with everything" dont sit quite right with me, but I'll give it some thought as to what to change this to